tests/release/e2e-test-push-image-to-pyxis.go

```diff
@@ -28,9 +34,14 @@ var _ = framework.ReleaseSuiteDescribe("[HACBS-1571]test-release-e2e-push-image-
 	var kubeController tekton.KubeController
 
 	var devNamespace, managedNamespace string
+	var imageIDs []string
+	var pyxisKeyDecoded, pyxisCertDecoded []byte
+	var buildPrName, additionalBuildPrName, releasePrName, additionalReleasePrName string
 
 	BeforeAll(func() {
-		fw, err = framework.NewFramework("release-e2e-pyxis")
```

Comment:
I've renames this in order to  :
1- Making the name be short to avoid failing on length of string
2-Name is generated with this prefix and will be added 4 more letters that will be randomly set
to avoid an issue that I hit when test fails and userSignUp  and namespace would be exist , if it's same name then the toolchain will keep creating the namespace  and won't be bale to rerun tests again 

names been set in a way with name of specific service and test to be easier to debug in case of failures

Reply:
OK, thanks. Sounds good.